### ISSUE TITLE
Export button missing on new Product Page V2

### ISSUE THREAD

**Issue by MatShir** [PrestaShop/PrestaShop#35485]
### Discussed in https://github.com/PrestaShop/PrestaShop/discussions/35484

<div type='discussions-op-text'>

<sup>Originally posted by **ChronoBrake** February 26, 2024</sup>
Hey, where i can find "export" button on the new product page v2 list? It's looks like is missing. How i can add it manually?</div>

**Comment by MatShir** [comment]
@ChronoBrake Does exporting through the SQL manager not work?

**Comment by OrionCodeDev** [comment]
> @ChronoBrake Does exporting through the SQL manager not work?

Not really, in old products page list button "export" be really helpful to insta export data to CSV/excel. Many companies do that. How i can get export button back on the new page?

**Comment by OrionCodeDev** [comment]
The absence of an "Export" button on the new Product Page V2 in PrestaShop presents a significant workflow interruption for many businesses that rely on exporting data for various operational needs. The ability to instantly export data to CSV or Excel formats directly from the product list is a critical feature for several reasons:

1. **Consistency Across Interfaces**: Given that the export feature exists on other entity lists such as orders and customers, it's a noticeable omission on the product list page. Users expect a consistent experience across the platform, and this inconsistency can lead to confusion and reduce the efficiency of administrative tasks.

2. **Operational Efficiency**: The export button serves as a quick-access tool that streamlines the process of data manipulation and analysis. Without it, users must resort to more time-consuming methods such as manually collating data or using the SQL manager, which may not be as user-friendly or accessible for all users, especially those with less technical expertise.

3. **Data Integration**: Many businesses use the export feature to integrate their product data with other systems, such as inventory management, accounting software, or external databases. The ability to quickly export a CSV file can be crucial for maintaining up-to-date records across different platforms.

4. **Reporting and Analytics**: Exporting data allows businesses to perform advanced analytics using external tools. This data is often used for reporting, forecasting, and market analysis. Without an export button, the process of extracting this data becomes cumbersome and prone to errors, especially when dealing with large datasets.

5. **Customization and Flexibility**: The old version of the product list allowed for customized exports based on the filtered search or full product list, providing flexibility in how data is extracted and used. This level of customization should be a staple feature in any new version of the product list.

6. **User Expectation and Workflow**: Users who have integrated the export feature into their daily workflow may find the new version of the product list lacking. It disrupts their established processes and forces them to look for workarounds, which can be frustrating and time-consuming.

7. **Feature Parity**: When upgrading to a new version of any software, there is an expectation that previous features will be retained, if not improved upon. The absence of such a fundamental feature can be seen as a step backward, rather than an improvement.

In conclusion, the "Export" button is not just a convenience; it is a necessity for many businesses that manage their operations through PrestaShop. Its inclusion in the new Product Page V2 should be a priority to maintain the platform's utility, ensure user satisfaction, and keep up with the operational demands of modern e-commerce businesses. It is vital that this feature be reinstated to preserve the efficiency and functionality that users have come to rely on.

**Comment by MatShir** [comment]
In the meantime, could try to press the button "Export to SQL Manager"
<img width="255" alt="Capture d’écran 2024-02-26 à 18 42 37" src="https://github.com/PrestaShop/PrestaShop/assets/43613217/fb450b79-02bc-4437-b1ec-d993493a95dd">
Then press on "Save"
And on export on the SQL query list.
It should do the job. 
I understand the need, but there is a workaround from the meantime.

**Comment by OrionCodeDev** [comment]
@MatShir i found missing functions. I pushing merge request for fix this issue. #35488

**Comment by matks** [comment]
Fixed by https://github.com/PrestaShop/PrestaShop/pull/35488
